README.md

```diff
@@ -860,6 +860,15 @@ word-break:break-word
      </td>
      <td>Read</td>
    </tr>
+  <tr>
```

Comment:
Please remove - as this is the first documentation of the flag it should appear in README-template.md only (also,  it's current default vale is `false`)